### DISCUSSION TITLE
is there a way to not only claim squared shape

### DISCUSSION THREAD

**Discussion by Tizooo** [GriefPrevention/GriefPrevention#1476] [Q&A]
Hey is it possible to somehow not only do square shaped claims but somehow also L shaped claims since we want to do a city but there are some claims we would need to remove because we can not claim only a small piece

**Comment by carmeops** [comment]
agree to this, because it's annoying when your claim is odd shaped, you have to do many small claims, it's tedious

you could do an option for poly selection like in world edit, where you draw the shape one corner after another with a limit on how many corners (like, a L would use 6 corners, and an hollowed square would take 10 corners)

either that, or being able to merge claims, so you can deal with their trust and flags all at once as if they were a single claim
